### DISCUSSION TITLE
php artisan livewire:cache

### DISCUSSION THREAD

**Discussion by iateadonut** [livewire/livewire#5510] [3. Feature ideas]
Does livewire need to be cached on the fly?

If so, shouldn't that be done in storage rather than bootstrap/cache?

If not, is there a command, something like 'php artisan livewire:cache' that can cache all livewire components?

**Comment by iateadonut** [comment]
The file bootstrap/cache/laravel-components.php is not created by `php artisan view:cache`.  I'm not sure how it's made, but it seems to be made on the first call to a livewire component.

In our use case, we wanted all files possible generated and saved to an AMI when we do auto-scaling on aws.

**Comment by joshhanley** [comment]
@iateadonut I think the file is actually called `livewire-components.php` and yeah that's Livewire's component auto-discovery manifest.

You can generate it using `php artisan livewire:discover` so you can ensure it gets deployed. But it's not essential, Livewire updates the manifest automatically as needed regardless of if it exists in your scaling image or not, and it doesn't really speed up the response time. So just personal preference if you want to use it.

Hope this helps!
